.prow.yaml

```diff
@@ -119,7 +119,7 @@ presubmits:
             items:
               - key: satoken
                 path: satoken
-  - name: e2e-ubuntu-focal
+  - name: e2e-free5gc-ubuntu-focal
```

Comment:
@electrocucaracha shouldn't we make it generic? rather than calling it free5GC something like r1 or r2? Another thing can we use Jammy rather than Focal for OAI? Focal will work too but I would like to use the latest Ubuntu or Fedora.